### DISCUSSION TITLE
When will 2.0b release?

### DISCUSSION THREAD

**Discussion by pintermor9** [Rapptz/discord.py#7775] [General] [answered]
When will a beta release?

**Comment by scrazzz** [comment, marked as answer]
There's never been beta version afaik.
There's no ETA for the release of 2.0 on PyPi.

**Reply by pintermor9** [reply]
is there an eta for 2.0 now?

**Reply by LeoCx1000** [reply]
> is there an eta for 2.0 now? 

No
